.github/workflows/lint.yml

```
@@ -1,24 +1,17 @@
 on:
   push:
-    branches:
-      - main
-
+    branches: [ main, develop ]
   pull_request:
-    branches:
-      - main
-
+    branches: [ main, develop ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)